src/py/migraphx_py.cpp

```diff
@@ -581,6 +582,15 @@ MIGRAPHX_PYBIND11_MODULE(migraphx, m)
           py::arg("t"),
           py::arg("calibration") = std::vector<migraphx::parameter_map>{},
           py::arg("ins_names")   = std::unordered_set<std::string>{"dot", "convolution"});
+    m.def(
+        "autocast_fp8",
+        [](migraphx::program& p) {
+            migraphx::autocast_fp8_pass pass;
+            migraphx::module& input_module = *p.get_main_module();
+            pass.apply(input_module);
```

Comment:
Use `run_passes` to run the pass: `run_passes(*p.get_main_module(), autocast_fp8_pass{})`.

Reply:
Done.